install_template/config.yaml

```diff
@@ -575,3 +575,41 @@ products:
       - name: SLES 15
         arch: ppc64le
         supported versions: [8]
+  - name: Replication Server
+    platforms:
+#       - name: CentOS 7
+#         arch: x86_64
+#         supported versions: [7, 8]
+#       - name: CentOS 7
+#         arch: ppc64le
+#         supported versions: [7, 8]
+#       - name: AlmaLinux 8 or Rocky Linux 8
+#         arch: x86_64
+#         supported versions: [7, 8]
+#       - name: RHEL 7 or OL 7
+#         arch: x86_64
+#         supported versions: [7, 8]
+#       - name: RHEL 7 or OL 7
+#         arch: ppc64le
+#         supported versions: [7, 8]
+#       - name: RHEL 8 or OL 8
+#         arch: x86_64
+#         supported versions: [7, 8]
+#       - name: Ubuntu 18.04 or Debian 9
+#         arch: x86_64
+#         supported versions: [7, 8]
+#       - name: Ubuntu 20.04 or Debian 10
+#         arch: x86_64
```

Comment:
Let's try & leave everything uncommented in this file going forward - we can restrict what is deployed via rules in deploy.mjs

Reply:
Fixed in latest commit